docs/README.md

```diff
@@ -1,52 +1,68 @@
-
 # Introduction
 
-This document is for Apostrophe 2.x developers who wish to try out Apostrophe 3.x and give feedback. We look forward to your input!
+This document assumes a level of familiarity with Apostrophe 2. We suggest building your production projects with the latest stable release of [Apostrophe 2](https://github.com/apostrophecms/apostrophe). This repository reflects our current progress towards a stable 3.0 release, which is scheduled in Q1 of 2021. We're eager to share our progress so far and grateful for any feedback you might have.
 
-## Getting Started
+### New in Apostrophe 3
 
-Grab the boilerplate project and give your project a name of its own. We're assuming `myproject` here. Replace that with your own project name wherever you see it. Use a short name with only letters, digits and dashes.
+- New editing experience
+- lots of cool headless stuff
+- rewritten in vue
+- module architecture
+- etc
 
-```
-git clone https://github.com/apostrophecms/a3-boilerplate myproject
-cd myproject
-npm install
-node app @apostrophecms/user:add admin
-npm run dev
-```
+### Coming Soon in Future Releases
 
-> Be sure to give the admin user a password when prompted.
+- Permissions (all admins)
+- Version History
+- Image Cropping
+- Batch Operations
+- User Roles and Groups
 
-Now you can [view the site here on your own computer](http://localhost:3000).
+## Prerequisites
 
-To edit the site, [log in here](http://localhost:3000/login). Click on the text on the page to start editing. **Be sure to try selecting the text;** you have many formatting options which appear only when text is selected.
+Apostrophe 3 introduces a number of new features and APIs, but the server requirements are the same as before. This document assumes you're running on macOS**.** As always, we recommend installing the following with [Homebrew](https://brew.sh/).
 
-> You don't have to "save" anything because your changes are saved right away. However we plan to introduce a more intentional save button in alpha 2.
+| Sotfware | Minimum Version |
+| ------------- | ------------- |
+| Git  ||
+| Xcode  ||
+| Node.js | 10.x |
+| npm  | 6.x  |
+| MongoDB  | 3.6  |
+| Imagemagick (optional)  ||
 
-## Making your project your own
+::: tip Note: 
+When you make code changes the boilerplate project will automatically restart and refresh the browser. If you get a "port in use" error, press control-C and start `npm run dev` again. We're tracking down how to reliably reproduce this issue.* 
+:::
 
-First **edit `app.js` and change `shortName` to `myproject`** (replace with the folder name of your own project).
+## Getting Started
 
-Then, to push your project to your own github, cut it loose from ours:
+Clone the Apostrophe 3 Boilerplate project give your project a name of its own. Legal characters consist of letters, digits and dashes. We're assuming **myproject** here. Be sure to give the admin a user and password when prompted. 
```

Comment:
Two sentences ran together here.

I see you want to skip talking about how to get this project into their own repo. I think it's nice, but it's not essential for now.